Cargo.toml

```diff
@@ -36,17 +36,16 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cw0 = {  version = "0.10.0" }
-cw2 = { version = "0.11" }
-cw20 = { version = "0.10.0" }
-cw20-base = { version = "0.10.0", features = ["library"] }
-cosmwasm-std = { version = "1.0.0-beta" }
-cosmwasm-storage = { version = "1.0.0-beta" }
-cw-storage-plus = "0.10"
-schemars = "0.8.3"
-serde = { version = "1.0.127", default-features = false, features = ["derive"] }
-thiserror = { version = "1.0.26" }
+cw0 = "0.10.3"
+cw2 = "0.15.0"
+cw20 = "0.15.0"
+cosmwasm-schema = "1.1.2"
+cw20-base = { version = "0.15.0", features = ["library"] }
+cosmwasm-std = { version = "1.1.2" }
+cosmwasm-storage = { version = "1.1.2" }
+cw-storage-plus = "0.15.0"
+schemars = "0.8.10"
+serde = { version = "1.0.144", default-features = false, features = ["derive"] }
+thiserror = { version = "1.0.35" }
 
-[dev-dependencies]
```

Comment:
probably still want the `[dev-dependencies]` key :)

Reply:
yes my bad:  24b7c30f07ce578f9317f8e7539be88a7cb44263